gettsim/tests/test_kindergeld.py

```diff
@@ -1,26 +1,32 @@
 import numpy as np
+import pandas as pd
 import pytest
 from pandas.testing import assert_series_equal
 
+from gettsim.config import ROOT_DIR
 from gettsim.taxes.kindergeld import kg_eligibility_hours
 from gettsim.taxes.kindergeld import kg_eligibility_wage
 from gettsim.taxes.kindergeld import kindergeld
-from gettsim.tests.auxiliary_test_tax import get_policies_for_date
-from gettsim.tests.auxiliary_test_tax import load_tax_benefit_data
-from gettsim.tests.auxiliary_test_tax import load_test_data
+from gettsim.tests.policy_for_date import get_policies_for_date
 
 
 input_cols = ["hid", "tu_id", "pid", "age", "w_hours", "ineducation", "m_wage"]
```

Comment:
The convention is to capitalize module variables.

Reply:
Done